active/InformationBar/InformationBar.md

```diff
@@ -398,6 +398,62 @@ Please view the guidance for
 for more information about text localization in your notification.
 
 
+# InfoBarPanel class
+
+A layout panel that positions its children horizontally if there’s available space,
+otherwise positions them vertically. This panel is intended to only be used as part
+of the ControlTemplate of the InfoBar control.
+
+## InfoBar Padding properties
+
+The `PaddingInHorizontalOrientation` property gets/sets the distance between the edges
+of the panel and its children, when the panel is laying out items horizontally.
+The `PaddingInVerticalOrientation` property does likewise when the panel is layout out
+items vertically.
+
+[API note: these padding properties are analogous to ex the 
+[StackPanel.Padding](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.StackPanel.Padding)
+property.]
+
+## InfoBar Spacing attached properties
+
+The `SpacingInHorizontalOrientation` attached property gets/sets the distance between i tems
+when the panel is laying out items in the horizontal orientation. Similarly 
+`SpacingInVerticalOrientation` for vertical layout. This property is set on the child items,
+so the spacing can be different between different item pairs.
+
+[API note: these spacing properties are analogous to ex the
+[StackLayout.Spacing](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.StackLayout.Spacing)
+property, except they're attached properties so that they can be set on the panel children.]
+
+Note that if adjacent children both have non-zero spacing set, the values will be added
+together. That is if a left child has right spacing, and a right child has left spacing,
+the total spacing will be the sum of the two values.
```

Comment:
OK here is the answer :). It really doesn't sound like `Spacing` then. `Spacing` is a `double` and a simple concept, but this is a `Thickness` and is more complex.

Reply:
This also doesn't describe how the other values of the Thickness are used (for example how the height is used when it's  in a horizontal layout).

Reply:
Discussion: Agreed, this is now a margin (see earlier comment).